### ISSUE TITLE
Data Wrangling Cheat Sheet: colors for reshaping illustrations

### ISSUE THREAD

**Issue by sijow** [tk3369/www.ahsmart.com#8]
I love this cheat sheet, thanks for making it!

A small suggestion: I think the "Reshape Data - changing layout" section would be easier to understand with different colors for the different variables. I actually didn't understand the illustrations at first, then I read the [pandas cheat sheet](https://pandas.pydata.org/Pandas_Cheat_Sheet.pdf) and immediately got it 😊

**Comment by tk3369** [comment]
Good idea. I can probably tackle this when I update the cheat sheet again later.
